via/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+def includeme(_config):  # pragma: no cover
+    pass
```

Comment:
This needs to be here so that `app.py` can `config.include("via.models")`. The next PR (https://github.com/hypothesis/via/pull/1072) will actually add a model here